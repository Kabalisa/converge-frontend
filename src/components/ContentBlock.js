import React from 'react';

const ContentBlock = () => (
  <div className="content-block">Content Block</div>
);

export default ContentBlock;
