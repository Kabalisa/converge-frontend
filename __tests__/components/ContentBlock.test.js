import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import ContentBlock from '../../src/components/ContentBlock';

describe('ContentBlock Component', () => {
  const wrapper = shallow(<ContentBlock />);

  it('Should render correctly', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ContentBlock />, div);
  });
  it('renders correctly in memory', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
