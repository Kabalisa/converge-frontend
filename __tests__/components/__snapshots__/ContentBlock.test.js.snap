// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentBlock Component renders correctly in memory 1`] = `
<div
  className="content-block"
>
  Content Block
</div>
`;
